webapp/packages/core-root/src/SessionResource.ts

```diff
@@ -95,6 +93,23 @@ export class SessionResource extends CachedDataResource<SessionState | null> {
     return session;
   }
 
+  async touchSession() {
+    if (!this.data?.valid) {
+      return;
+    }
+
+    const valid = await this.graphQLService.sdk.touchSession();
+
+    if (!valid) {
+      this.setData({
+        ...this.data,
+        valid,
+      });
+    }
```

Comment:
how we will now how many time left to session expire? (we need to get new `remainingTime` value because `touchSession` will adjust it)